README.md

```diff
@@ -84,6 +92,16 @@ This step is required on **Linux** and is required to create eve bootable images
 ```sh
 $ docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 ```
+<details><summary>M1 Mac cross builds</summary>
+
+For docker to emulate a chip architecture different from the build host, perform this additional step:
```

Comment:
Is this necessary? The previous step - `multiarch/qemu-user-static` - is meant to provide qemu user static. As for `binfmt`, docker desktop ships with binfmt already run.